src/token-processor/util/sip-validation.ts

```diff
@@ -361,17 +370,25 @@ export function getContractLogMetadataUpdateNotification(
   }
 }
 
+export type NftMintEvent = {
+  contractId: string;
+  tokenId: bigint;
```

Comment:
There are a few areas where `token-id` int sizes are different, like in some postgres tables it's an int32. Anything to worry about?

Reply:
Good catch, I'll make a pass on these to double check